rulette-core/src/main/java/com/github/kislayverma/rulette/core/util/RuletteInputProcessor.java

```diff
@@ -26,11 +33,16 @@
             if(fieldValue == null){
                 fieldValueString = "";
             }else {
-                fieldValueString = fieldValue.toString();
+                if(fieldValue instanceof Date){
+                    fieldValueString = new DateTime(fieldValue).toString(formatter);
+                }else {
+                    fieldValueString = fieldValue.toString();
+                }
             }
```

Comment:
What does this change do?

Reply:
It reads Date in the particular format we have specified, as date can be sent in any format in the request payload.